Documentation/networking.md

```diff
@@ -13,6 +13,33 @@ When `--net=host` is passed the pod's apps will inherit the network namespace of
 If rkt is directly called from the host the apps within the pod will share the network stack and the interfaces with the host machine.
 This means that every network service that runs in the pod has the same connectivity as if it was started on the host directly.
 
+The network namespaces of a host process and a pod process can be compared directly by reading a link in the *procfs*.
+
+```sh
+$ readlink /proc/self/ns/net
+net:[4026531969]
+$ sudo rkt run --interactive --net=host busybox-1.23.2-linux-amd64.aci --exec /bin/busybox -- readlink /proc/self/ns/net
```

Comment:
I uploaded busybox image with a port in `kinvolk.io/aci/busybox:1.24`, maybe you can use that.